### PR TITLE
[Editor] Allow to undo/redo committed text modifications for FreeText

### PR DESCRIPTION
**Pull request by calixteman**

### CODE REVIEW

File: src/display/editor/freetext.js [hunk 1]
```diff
@@ -355,8 +355,26 @@ class FreeTextEditor extends AnnotationEditor {
     }
 
     this.disableEditMode();
-    this.#content = this.#extractText().trimEnd();
+    const savedText = this.#content;
+    const newText = (this.#content = this.#extractText().trimEnd());
+    if (savedText === newText) {
+      return;
```
Comment (Snuffleupagus) [inline src/display/editor/freetext.js:361]: Previously we *unconditionally* called `this.#setEditorDimensions();` however that no longer happens, was that intended?

Comment (calixteman) [reply src/display/editor/freetext.js:361]: this method is used to set the dimensions in percent of the editor so if its content  didn't change, the dimensions should be exactly the same as before so no need to set them again.
To answer to your question: yes it was.

**Review by Snuffleupagus — APPROVED** [review]
r=me, thank you!

**Pull request merged**